### DISCUSSION TITLE
调用二维码登录接口但未返回二维码图片的base64编码

### DISCUSSION THREAD

**Discussion by Euphoriahz** [Binaryify/NeteaseCloudMusicApi#1719] [Q&A] [answered]
我使用/login/qr/key接口生成key之后调用二维码生成接口获取二维码信息，这里请求地址中加入了qrimg参数来获取二维码图片的base64编码信息，但是返回的数据中qrimg为空
<img width="450" alt="微信图片_20230315155651" src="https://user-images.githubusercontent.com/93807619/225243745-120240d6-ca60-4aee-b9f6-bfe3ce6c8d25.png">
<img width="941" alt="微信图片_20230315155658" src="https://user-images.githubusercontent.com/93807619/225243777-95c315e9-7169-41ad-8407-7832781770bd.png">

**Comment by Binaryify** [comment, marked as answer]
&qrimg=1

**Reply by Euphoriahz** [reply]
感谢感谢
